ILSpy.AddIn/ILSpy.AddIn.csproj

```diff
@@ -147,6 +148,7 @@
     <Compile Include="ILSpyAddInPackage.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="PkgCmdID.cs" />
+    <None Include="Samples\ILSpyAddInSamples.cs" />
```

Comment:
Samples provided for reference only, not compiled or copied to the build output.